### PR TITLE
BOTMETA.yml: Remove vfauth as etcd3 maintainer

### PR DESCRIPTION
**Pull request by vfauth**
##### SUMMARY
I don't want to get notifications anymore.

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
Module etcd3

### CODE REVIEW

File: .github/BOTMETA.yml [hunk 1]
```diff
@@ -294,7 +294,7 @@ files:
   $modules/clustering/consul/:
     maintainers: $team_consul
   $modules/clustering/etcd3.py:
-    maintainers: evrardjp vfauth
+    maintainers: evrardjp
```
Comment (felixfontein) [inline .github/BOTMETA.yml:297]:
```suggestion
    maintainers: evrardjp
    ignore: vfauth
```

**Review by felixfontein — COMMENTED** [review]
@vfauth thanks for your work on the etcd3 module!

Unfortunately just removing yourself form the maintainer list won't do the job, since you're listed as the module's author. The bot is using that field as well... To make sure you no longer get notifications, you need to add yourself to the ignore list, see my suggestion below:

**Review by felixfontein — APPROVED** [review]

**Comment by vfauth** [comment]
Thank you for the help @felixfontein ! That new commit should do the job.

**Comment by felixfontein** [comment]
@vfauth thanks! Please note that the bot needs a bit of time to update it's internal cache of the botmeta file, it should not take longer than a day though. (TBH I have no idea whether the current bot still caches it and for how long ;) )

**Pull request merged**